### PR TITLE
Corrected English name of nb language.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**
See for example http://www.loc.gov/standards/iso639-2/php/code_list.php , where the name is listed as "Norwegian Bokmål".

### CODE REVIEW

**Comment by timgraham** [comment]
buildbot, test this please.

**Comment by timgraham** [comment]
Looks like you need to add `# -*- coding: utf-8 -*-` to this file for Python 2. Even after that, there's a test failure:
```
======================================================================
ERROR: test_i18n12 (template_tests.syntax_tests.test_i18n.I18nTagTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/case.py", line 331, in run
    testMethod()
  File "/home/tim/code/django/django/test/utils.py", line 208, in inner
    return func(*args, **kwargs)
  File "/home/tim/code/django/tests/template_tests/utils.py", line 61, in inner
    func(self)
  File "/home/tim/code/django/tests/template_tests/syntax_tests/test_i18n.py", line 149, in test_i18n12
    output = self.engine.render_to_string('i18n12')
  File "/home/tim/code/django/django/template/engine.py", line 180, in render_to_string
    return t.render(Context(context))
  File "/home/tim/code/django/django/template/base.py", line 207, in render
    return self._render(context)
  File "/home/tim/code/django/django/test/utils.py", line 94, in instrumented_test_render
    return self.nodelist.render(context)
  File "/home/tim/code/django/django/template/base.py", line 976, in render
    bit = node.render_annotated(context)
  File "/home/tim/code/django/django/template/base.py", line 943, in render_annotated
    return self.render(context)
  File "/home/tim/code/django/django/templatetags/i18n.py", line 20, in render
    context[self.variable] = [(k, translation.ugettext(v)) for k, v in settings.LANGUAGES]
  File "/home/tim/code/django/django/utils/translation/__init__.py", line 85, in ugettext
    return _trans.ugettext(message)
  File "/home/tim/code/django/django/utils/translation/trans_real.py", line 321, in ugettext
    return do_translate(message, 'ugettext')
  File "/home/tim/code/django/django/utils/translation/trans_real.py", line 301, in do_translate
    result = getattr(translation_object, translation_function)(eol_message)
  File "/usr/lib/python2.7/gettext.py", line 404, in ugettext
    return unicode(message)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 14: ordinal not in range(128)
```

**Comment by petterreinholdtsen** [comment]
[Tim Graham]
> Looks like you need to add `# -*- coding: utf-8 -*-` to this file for
> Python 2. Even after that, there's a test failure:

Right.  I guess UTF-8 and Python 2 is always going to be a pain.

So, what should be done about this, and by whom?  I am not capable of
rewriting the code base to work properly with non-ascii language names.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by timgraham** [comment]
I created [a ticket](https://code.djangoproject.com/ticket/26103#ticket) for the issue and will close this pull request for now.

File: django/conf/global_settings.py [hunk 1]
```diff
@@ -104,7 +104,7 @@
     ('mn', gettext_noop('Mongolian')),
     ('mr', gettext_noop('Marathi')),
     ('my', gettext_noop('Burmese')),
-    ('nb', gettext_noop('Norwegian Bokmal')),
+    ('nb', gettext_noop('Norwegian Bokmål')),
```
Comment (claudep) [inline django/conf/global_settings.py]: The correct syntax is `u'Norwegian Bokmål'` (unless we import `from __future__ import unicode_literals` at the top). It's also fine with Python 3 if we don't have to support Python < 3.3.

**Comment by timgraham** [comment]
Petter, could you try updating with `encoding` and `unicode_literals`?

**Comment by petterreinholdtsen** [comment]
[Tim Graham]
> Petter, could you try updating with `encoding` and `unicode_literals`?

Not quite sure I understood what you meant, but I've tried.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by timgraham** [comment]
buildbot, test this please.

**Comment by timgraham** [comment]
merged in a1fba4e843901eb35820c14d83a5d7d5622432fe, thanks!

**Pull request closed without merging**